messaging/src/main/java/org/axonframework/queryhandling/duplication/DuplicateQueryHandlerResolution.java

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2010-2019. Axon Framework
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.axonframework.queryhandling.duplication;
+
+/**
+ * Enumeration describing a set of reasonable {@link DuplicateQueryHandlerResolver} implementations. Can be used to
+ * configure how a {@link org.axonframework.queryhandling.QueryBus} should react upon a duplicate subscription of a
+ * query handling function.
+ *
+ * @author Mitchell Herrijgers
+ * @since 4.6
```

Comment:
The plan is to add a `0` from now on for the first minor release. So that the schema constantly is "x.y.z.," with no jump to "x.y.".
So, the since tag should point to 4.6.0 :-)